src/settings/FileExtensions/FileExtensionForm.test.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import faker from 'faker';
+import { fireEvent } from '@testing-library/react';
+import { noop } from 'lodash';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../test/jest/__mock__';
+import {
+  renderWithFinalForm,
+  translationsProperties,
+} from '../../../test/jest/helpers';
+
+import { FileExtensionFormComponent } from './FileExtensionForm';
+
+const initialValuesNewForm = {
+  dataTypes: [],
+  description: '',
+  extension: '',
+  importBlocked: false,
+};
+const initialValuesEditForm = {
+  ...initialValuesNewForm,
+  id: faker.random.uuid(),
+  extension: '.dat',
+};
+
+const handleFormSubmit = jest.fn(() => Promise.resolve({}));
+
+const fileExtensionFormProps = {
+  pristine: false,
+  submitting: false,
+  form: {
+    getState: () => ({ values: { importBlocked: false } }),
+    change: noop,
+    reset: noop,
+  },
+  match: { path: '/settings/data-import/file-extensions' },
+  transitionToParams: noop,
+  handleSubmit: handleFormSubmit,
+  onCancel: noop,
+};
+
+const renderFileExtensionForm = ({
+  pristine,
+  submitting,
+  form,
+  initialValues,
+  handleSubmit,
+  onCancel,
+  transitionToParams,
+  match,
+}) => {
+  const component = () => (
+    <Router>
+      <FileExtensionFormComponent
+        pristine={pristine}
+        submitting={submitting}
+        form={form}
+        match={match}
+        initialValues={initialValues}
+        handleSubmit={handleSubmit}
+        onCancel={onCancel}
```

Comment:
```suggestion
        onCancel={noop}
```
If this prop doesn't change from test to test, why don't you add **noop** directly in the props